### ISSUE TITLE
Always in Login ID Window when start Lens Desktop on Linux

### ISSUE THREAD

**Issue by mengzyou** [lensapp/lens#5528]
**Describe the bug**
On Linux (OpenSUSE), can not start Lens, it always stays in "Login ID window".  

**To Reproduce**
1. Run Lens `./Lens-5.5.2-latest.x86_64.AppImage` .    

2. Open below window  
![pic_1](https://owncloud.mengz.dev/index.php/apps/files_sharing/ajax/publicpreview.php?x=1892&y=656&a=true&file=Screenshot_Lens_1st.png&t=6bCNyCuVDCDz6XJ&scalingup=0)    

3. Then show below window  
![pic_2](https://owncloud.mengz.dev/index.php/apps/files_sharing/ajax/publicpreview.php?x=1892&y=656&a=true&file=Screenshot_Lens_2nd.png&t=6hv4OkSZCldLuF8&scalingup=0)  

4. After input username  and password, it show  
![pic_3](https://owncloud.mengz.dev/index.php/apps/files_sharing/ajax/publicpreview.php?x=1892&y=656&a=true&file=Screenshot_Lens_3rd.png&t=UETpeHeCyVK3FuT&scalingup=0)  

5. Click `Go to Lens`, there was no effect, then I close this window, and right-click system tray icon > Open Lens, it open window from about Pic 1 again, till Pic_3 `Login ID` window, again an again, so I can not open the main window after logined in.
![pic_4](https://owncloud.mengz.dev/index.php/apps/files_sharing/ajax/publicpreview.php?x=1892&y=656&a=true&file=Screenshot_Lens_4th.png&t=tlvfNMdcRaJciom&scalingup=0)  

**Expected behavior**
Open the main window (not always login).

**Screenshots**
See above.

**Environment (please complete the following information):**
- Lens Version: 5.2.2-latest.x86_64
- OS: openSUSE Leap 15.3 with KDE-Plasma 5.18.6
- Installation method (e.g. snap or AppImage in Linux): AppImage in Linux

**Comment by Nokel81** [comment]
Interesting, I was unable to reproduce this on Ubuntu 20.04 with KDE Plasma 5.18.8!

**Comment by mengzyou** [comment]
Yes, it's strange, I remember it can skip Login to open the main window in older version. But for the new version it always require Login.

**Comment by deissnerk** [comment]
I have the same problem on MacOS. I used `brew` for installation and it downloaded `https://api.k8slens.dev/binaries/Lens-5.5.2-latest.20220530.1.dmg`

**Comment by deissnerk** [comment]
I found this file:
`~/Library/Application Support/Lens/extension-store/lenscloud-lens-extension/startup-store.json`

It contains:
```
{"skipSignup":true}
```

**Comment by deissnerk** [comment]
Seems to be a duplicate to #5444

**Comment by iconoeugen** [comment]
The authentication seems to be required to get access to Lens Spaces which seems to be unnecessary for the users complaining here about it 🤔

My working solution was just to go the the installation path of Lens and find the extensions dir resources/extensions and remove the lenscloud-lens-extension extension

**Comment by mengzyou** [comment]
Thanks for helping, and I found seems it can not write token to file *~/.config/Lens/extension-store/lenscloud-lens-extension/auth-token-store.json*,  the file content always  

```json
{
  "access_token": "",
  "id_token": "",
  "refresh_token": "",
  "refresh_trigger": 0
}
```

Finaly, I found a work around that copy this file from Windows (after login with my id), then open the Lens again, it show me the main window with login status.
